### PR TITLE
Enable the usbguard service in OSPP profiles.

### PR DESCRIPTION
**Pull request by adelton**
#### Description:

- Enable the usbguard service in OSPP profiles.

#### Rationale:

- Presumably it increases security.

### CODE REVIEW

File: linux_os/guide/services/usbguard/service_usbguard_enabled/rule.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Enable the USBGuard Service'
+
+description: |-
+    The USBGuard service should be enabled.
+    {{{ describe_service_enable(service="usbguard") }}}
+
+rationale: ""
```
Comment (jan-cerny) [inline linux_os/guide/services/usbguard/service_usbguard_enabled/rule.yml]: We typically explain why we suggest using the service. Check eg. rules service_chronyd_or_ntpd_enabled, service_rsyslog_enabled, service_firewalld_enabled for inspiration.

Comment (adelton) [reply linux_os/guide/services/usbguard/service_usbguard_enabled/rule.yml]: Well, I've checked `service_sssd_enabled` and that is very silent about the rationale ...

**Comment by adelton** [comment]
Added rationale, rebase on master -> 33286d992725c0ab2feafd280f0aa52ce5ca239c.

**Comment by jan-cerny** [comment]
@openscap-ci test this please

**Pull request merged**